lnst/RecipeCommon/Perf/Recipe.py

```diff
@@ -93,6 +94,20 @@ def add_measurement_results(
         )
         self._aggregated_results[measurement] = aggregated_results
 
+    """
+        Function returns end timestamp of warmup period and start of warm down period.
+        That results to slice measurement just for "interesting" part not including 
+        warm up and warm down periods.
+    """
+    def _get_real_flow_times(self, flows: List[BaseMeasurementResults]):
+
+        if isinstance(flows[-1], FlowMeasurementResults):
+            logging.info(f"Results alignment: Using times of flow measurement: {flows[-1]}")
+            return flows[-1].warmup_end, flows[-1].warmdown_start
+        else:
```

Comment:
As i mentioned bellow, this branch is for every other `MeasurementResults`, only `FlowMeasurementResults` class has implemented `warmup_end` and `warmdown_start` properties. Thats because we want to trim every result based on timestamps of last performed measurement. So in case last measurement wasn't the `FlowMeasurementResults` i've implemented mentioned branch.

Reply:
I think that within the context of Perf/Recipe.py this situation can't happen. So, probably raising an exception would make more sense.

I also suggest to add a comment before the if/else block explaining this. This is a "magic" number code and without knowing the background it's impossible to understand the code.

Reply:
`Perf/Recipe.py` is universal and this situation could imo happen in that context, not everyone has to be running flow measurements as the main ones, there could be an io measurement for example. But that's not important, we don't really do non flow measurements as the main topic and are unlikely to do so, which means we could remove that and deal with it later.

But having it this way won't actually break anything, and i think the code still makes sense.

`-1` in this case is **not** a magic number code... it's a standard way to refer to the "last in the list". If anything I'd say that we may want to document how the `PerfRecipe` works so that it's better understood that the measurements are started and stopped in order.